docker/development.yml

```diff
@@ -56,3 +56,11 @@ services:
       - 'traefik.http.routers.firebase_ui.rule=Host(`ui.firebase.localhost`)'
       - 'traefik.http.routers.firebase_ui.service=firebase_ui'
       - 'traefik.http.services.firebase_ui.loadbalancer.server.port=4000'
+
+  redis:
+    ports:
+      - '6379:6379'
+
+  elasticsearch:
+    ports:
+      - '9200:9200'
```

Comment:
This is fine, but just in case you didn't know, we have `http://elasticserach.localhost` which routes here via Traefik too.  Doing both is fine.

Reply:
oh, so maybe we should add that [here](https://github.com/Seneca-CDOT/telescope/blob/80d4293277e42b2915b95842f93115defa238a69/config/env.development#L168) (and then we won't need the changes in this PR).

Reply:
Good idea.  In development it can be this, and in staging/production we can use the docker container version.

Reply:
@humphd I think it's easier if you merge #1935 first, and then I'll change the url in `env.development` instead of exposing ports.